tasks/pipeline-dbconf.yml

```diff
@@ -40,17 +40,17 @@
     priv: "MCP.*:ALL"
     state: "present"
 
-- name: "Initialize database MCP"
+- name: "Load MySQL seed file"
   mysql_db:
     name: "MCP"
     state: "import"
     target: "{{ archivematica_src_dir }}/archivematica/src/MCPServer/share/mysql"
-  when: "createdb_result|changed"
+  when: "createdb_result|changed and archivematica_src_am_pre_migration|bool"
 
 # TODO: Does this need to run as root or user archivematica ??
-- name: "Invoke django syncdb"
+- name: "Run migrations"
   django_manage:
-    command: "syncdb"
+    command: "migrate"
```

Comment:
I think you're right, since newer Django wants `migrate` and older Django wants `syncdb`, even though they partly overlap in functionality.